test/test-poll.c

```diff
@@ -388,17 +388,17 @@ static void connection_poll_cb(uv_poll_t* handle, int status, int events) {
       new_events &= ~UV_WRITABLE;
     }
   }
-#ifndef _AIX
+#if !defined(_AIX) && !defined(__MVS__)
   if (events & UV_DISCONNECT) {
     context->got_disconnect = 1;
     ++disconnects;
     new_events &= ~UV_DISCONNECT;
   }
 
   if (context->got_fin && context->sent_fin && context->got_disconnect) {
-#else /* _AIX */
+#else /* _AIX  && __MVS */
```

Comment:
typo? this is the only instance on this patch where `__MVS` is used instead of `__MVS__` it seems.